### PR TITLE
Feature/last seen page

### PR DESCRIPTION
**Pull request by mrSelieznov**
#187 Task

Create last seens page

### CODE REVIEW

File: src/components/LastSeen.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react';
+import api from 'src/service/api';
+import RenderResults from './RenderResults';
+
+const LastSeen = () => {
+    const [data, setData] = useState([]);
```
Comment (solomiia-sahala) [inline src/components/LastSeen.tsx:6]: please, specify type of useState value with Typescript

File: src/components/LastSeen.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            setData(res);
+        });
+    }, [])
+
+    return (
+        <RenderResults list={data} />
+    );
+};
+
+export default LastSeen;
```
Comment (solomiia-sahala) [inline src/components/LastSeen.tsx:19]: pls, add line in the end

File: src/components/LastSeen.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react';
+import api from 'src/service/api';
+import RenderResults from './RenderResults';
+
+const LastSeen = () => {
+    const [data, setData] = useState([]);
+
+    useEffect(() => {
+        api.getDataByIds(api.getHistoryIdList()).then((res: any) => {
```
Comment (Kachyr) [inline src/components/LastSeen.tsx:9]: Does the response type have to be any?

File: src/components/LastSeen.tsx (hunk 2 above)
Comment (oleksandrstarov) [reply src/components/LastSeen.tsx:19]: It is hidden by default, if there is no line then red (-) will be shown

**Review by oleksandrstarov — APPROVED** [review]
Fix comments and merge

**Pull request merged**